privacy_partner_to_be_forgotten/readme/DESCRIPTION.md

```
@@ -0,0 +1,21 @@
+The module allows anonymizing contacts that represent natural persons (i.e., individuals) when they are no longer needed or when a data erasure request is received.
+
+This module provides a method to anonymize all personally identifiable information (PII) from a `res.partner` record:
+- Name (e.g., replaced with initials or a generic label) for individual contacts
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)